egs/librispeech/asr/nnlm/local/model.py

```diff
@@ -0,0 +1,154 @@
+import math
```

Comment:
We should get in the habit of acknowledging where we got files from, if they were copied from elsewhere...

Reply:
No problem. I will add a reference into every file. Now all references are together added in run.sh.